Models/UserFriendlyException.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace PSP.Models
+{
+    public class UserFriendlyException : Exception
```

Comment:
https://github.com/AusteZ/PSP/pull/3/files#r1444241491 same things as here, I think it would be good to add separate namespace for exceptions `PSP.Exceptions` or at least `PSP.Models.Exceptions`

Reply:
moved to PSP.Models.Exceptions